addon/helpers/pikaday.js

```diff
@@ -11,57 +10,61 @@ deprecate(
 );
 
 const openDatepicker = function(element) {
-  $(element).click();
+  document.querySelector(element).click();
 
   return PikadayInteractor;
 };
 
+const MONTH_SELECTOR = '.pika-lendar:visible .pika-select-month';
+const YEAR_SELECTOR = '.pika-lendar:visible .pika-select-year';
+
 const PikadayInteractor = {
-  selectorForMonthSelect: '.pika-lendar:visible .pika-select-month',
-  selectorForYearSelect: '.pika-lendar:visible .pika-select-year',
   selectDate(date) {
     const day = date.getDate();
     const month = date.getMonth();
     const year = date.getFullYear();
     const selectEvent = 'ontouchend' in document ? 'touchend' : 'mousedown';
 
-    $(this.selectorForYearSelect).val(year);
-    triggerNativeEvent($(this.selectorForYearSelect)[0], 'change');
-    $(this.selectorForMonthSelect).val(month);
-    triggerNativeEvent($(this.selectorForMonthSelect)[0], 'change');
+    const yearSelectorElements = document.querySelectorAll(YEAR_SELECTOR);
+    updateElementValues(yearSelectorElements, year);
+    triggerNativeEvent(yearSelectorElements[0], 'change');
+
+    const monthSelectorElements = document.querySelectorAll(MONTH_SELECTOR);
+    updateElementValues(monthSelectorElements, month);
+    triggerNativeEvent(monthSelectorElements[0], 'change');
 
     triggerNativeEvent(
-      $(
-        'td[data-day="' +
-          day +
-          '"]:not(.is-outside-current-month) button:visible'
-      )[0],
+      document.querySelector(
+        `td[data-day="'${day}'"]:not(.is-outside-current-month) button:visible}`
```

Comment:
I think some of these are jQuery specific extensions we'll need to refactor / rethink.
I tried this quickly in Chrome and found the following:
<img width="1065" alt="Screen Shot 2020-02-06 at 15 30 35" src="https://user-images.githubusercontent.com/472721/73976136-a9bacb00-48f5-11ea-9138-ee7b72af27d1.png">

Reply:
This error is visible with jQuery too. 🤔

Reply:
Something similar used here: https://github.com/adopted-ember-addons/ember-pikaday/blob/d0972f76a76dd1958cb625ad66f2109cc9dd2274/addon-test-support/interactor.js#L25